IdSSLOpenSSLHeaders.pas

```diff
@@ -19016,6 +19046,12 @@ function IsOpenSSL_DTLSv1_Available : Boolean;
     Assigned(DTLSv1_client_method);
 end;
 
+function IsOpenSSL_TLSv1_3_Available : Boolean;
+{$IFDEF USE_INLINE} inline; {$ENDIF}
+begin
+  Result := Assigned(TLS_method);

```

Comment:
Probably need to look for `TLS_client_method()`  and `TLS_server_method()` as well, like the other `XXX_Available()` functions are doing.